docs/streams/upgrade-guide.html

```diff
@@ -91,6 +91,20 @@ <h3><a id="streams_api_changes_280" href="#streams_api_changes_280">Streams API
         We extended <code>StreamJoined</code> to include the options <code>withLoggingEnabled()</code> and <code>withLoggingDisabled()</code> in
         <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-689%3A+Extend+%60StreamJoined%60+to+allow+more+store+configs">KIP-689</a>.
     </p>
+    <p>
+        We add and option to <code>addThread()</code> and <code>removeThread()</code> in
```

Comment:
`We add and option to` ?

-> `We added two new method to `KafkaStreams`, namely <code>addThread()</code> and <code>removeThread()</code> ...`